core/README.md

```diff
@@ -25,5 +26,16 @@ poetry install
 poetry run python -m explorer.extract --help
 
 # Extract the Nix data for nixpkgs on x86_64-linux distributions
-poetry run python -m explorer.extract
+poetry run python -m explorer.extract outfile.json
+```
+
+To develop, install the dependencies, and feed them to [vscode](https://code.visualstudio.com/)
+or your favorite editor:
+
+```bash
+# Install dependencies
+poetry install
+
+# Run vscode within poetry context:
+poetry run code .
```

Comment:
- `poetry install` has been mentionned earlier
- VSCode doesn't need to be run from the poetry shell, the Pylance can detect the poetry env used in a project

```suggestion
```